www-client/chromium/chromium-81.0.4044.113.ebuild

```diff
@@ -90,7 +90,7 @@ BDEPEND="
 	${PYTHON_DEPS}
 	>=app-arch/gzip-1.7
 	!arm? (
-		dev-lang/yasm
+		!ppc64? ( dev-lang/yasm )
```

Comment:
Let's change this to:
```
amd64? ( dev-lang/yasm )
x86? ( dev-lang/yasm )
```

Reply:
yeah current double nesting is ugly =)
isn't it needed on arm64 as well?

Reply:
> isn't it needed on arm64 as well?

Maybe? I thought yasm only supported amd64 and x86 output though.

Reply:
Based on this code search, I think it is only used for amd64 and x86.

https://source.chromium.org/search?q=yasm_assemble&sq=

Reply:
AFAICS its only used for building bundled libvpx or we do something wrong ... But yes, only amd64 and x86 (not sure if x86 builds)

Reply:
Based on @floppym 's codesearch yasm is only used for libvpx, ffmpeg (win only), openh264 (unbundled) and libaom (not enabled). So correct dep should be:
`!system-vpx (`
`amd64? (dev-lang/yasm)`
`x86? (dev-lang/yasm)`
`)`

Reply:
pushed corrected dep.

btw, should I update chromium-83 ebuild as well with this glue code?
I don't have patches for 83 so I can't actually test it

Reply:
> pushed corrected dep.
> 
> btw, should I update chromium-83 ebuild as well with this glue code?
> I don't have patches for 83 so I can't actually test it

Sure chromium-83 is masked anyway cause it has runtime crashes.